apps/frontend/src/network/users/__mocks__/api.ts

```diff
@@ -38,3 +39,22 @@ export const getUsers = jest.fn(
   async ({ pageSize }: GetListOptions): Promise<ListUserResponse> =>
     createListUserResponse(pageSize ?? 10),
 );
+
+export const getInstitutions = jest.fn(
```

Comment:
why bother define the default implementation if i may ask? I see you apply a custom resolve response anyway https://github.com/yldio/asap-hub/pull/940/files#diff-3108a0410b24cdad1ffe15c3d2688b42bfbfc211b3a79eca00b5f4e9f67d1078R77

Reply:
for the other tests or using the api somewhere else